fbw-a32nx/src/behavior/src/A32NX_Interior_EFIS.xml

```diff
@@ -17,7 +17,7 @@
 
         <UseTemplate Name="FBW_AIRBUS_Push_EFIS_Button_Template">
             <INDICATOR_CODE type="rnp" return="boolean">
-                (L:BTN_LS_#ID#_FILTER_ACTIVE, boolean)
+                (L:A32NX_BTN_LS_#ID#_FILTER_ACTIVE, boolean)
```

Comment:
Why is the LS button var changing in this PR?

Reply:
Rust adds the prefix A32NX to each  variable.

I need this variable in the Rust code to be able to listen to ILS ident. It's FCOM compliant.

Reply:
fbw-common/src/wasm/systems/systems/src/communications/mod.rs

Line 286

Reply:
Hmmm, I suspect this is not quite how it actually works. The FCOM is unfortunately not usually sufficient to implement things.

Reply:
How do you think it works? I can work on it if I have some references

Reply:
Not sure without doing a bit of research, but I don't think there's a direct bus from the FCU to the ACP.

Reply:
Yes you are definitely right that there's no direct bus but it's all I could figure out. 

Should I leave like it like this and add FIXME or comment it out?

Reply:
Leave it like this and add a FIXME. 👍